### DISCUSSION TITLE
Remember bar location

### DISCUSSION THREAD

**Discussion by SkyDrift** [valinet/ExplorerPatcher#367] [Ideas]
I use multi Monitors and set the taskbar vertical to the right on the second monitor,  but every time I restart  pc or refresh file explorer it moves back to the  right side of the Main monitor so was wounder if there was a way to remember the  second monitor X location state along with the all ready remembered orientation to be added to this ExplorerPatcher.

**Comment by valinet** [comment]
This has been discussed extensively in the past. tl;dr is the main taskbar has to sit on the main monitor, it is a restriction imposed by Windows Explorer itself. This has always been the intended behavior, it was a bug that it let you do that in Windows 10. In the mean time, Microsoft fixed this bug. Also, see the following threads, I tried to provide some mitigation at some point, did not work that well, more work is still required in that area:

https://github.com/valinet/ExplorerPatcher/issues/171
https://github.com/valinet/ExplorerPatcher/issues/14#issuecomment-936412362
